upgrade-operator/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "upgrade-operator"
```

Comment:
Should this package be operators? and then we have a an operator-upgrade binary?

Reply:
Yep, makes sense.